manifests/j/JTL/Wawi/1.5.47.2/JTL.Wawi.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: JTL.Wawi
+PackageVersion: 1.5.47.2
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral
+  InstallerType: inno
+  Scope: machine
+  InstallerUrl: https://downloads.jtl-software.de/setup-jtl-wawi_1.5.47.2_0719-0928_6a264dcd904.exe
+  InstallerSha256: 4AB1BCBA0C687BAF27C59BCA9C88A754822C0580866F87B784312AD2625D754A
+  InstallerSwitches:
+    Silent: /VERYSILENT
+    SilentWithProgress: /SILENT
```

Comment:
The InstallerSwitches is not needed as they get auto put in when the InstallerType is Inno/Nullsoft etc

Reply:
Oh okay